library/templates/dcp_censusdata.yml

```diff
@@ -0,0 +1,32 @@
+dataset:
+  name: &name dcp_censusdata
+  version: "{{ version }}"
+  acl: public-read
+  source:
+    script: *name
+    path: https://s-media.nyc.gov/agencies/dcp/assets/files/excel/data-tools/census/census2020/nyc_decennialcensusdata_2010_2020_change-core-geographies.xlsx
+    options:
+      - "AUTODETECT_TYPE=NO"
+      - "EMPTY_STRING_AS_NULL=YES"
+    geometry:
+      SRS: null
+      type: NONE
+
+  destination:
+    name: *name
+    geometry:
+      SRS: null
+      type: NONE
+    options:
+      - "OVERWRITE=YES"
+    fields: []
+    sql: null
+
+  info:
+    description: |
+      ###  Census
+      Decennial - high chance that 2030 will not follow same conventions
+      Currently has export with both data, 2010 or 2020 must be specified as version to determine which set of columns to use
+      This version does not contain data at the census block geography level
```

Comment:
if there's multiple geographic levels to this data, might be nice to replace this with something similar to what's in `_blocks.yml` like:

> This version contains data only at the city, boro, and __ level.

Reply:
Done!